rasa/utils/tensorflow/models.py

```diff
@@ -231,12 +251,12 @@ def _dynamic_signature(
         return [element_spec]
 
     def rasa_predict(
-        self, model_data: RasaModelData
+        self, batch_in: Tuple[np.ndarray]
```

Comment:
`Tuple[np.ndarray]` stands for a tuple containing [exactly one `np.ndarray`](https://stackoverflow.com/a/39458225/5157159), but I just noticed that we have this mistake at many places around the model code (in the context of batches), so I'm not sure what's the best thing to do -- to change it everywhere now or to create an issue...

Here, I assume that `batch_in` can contain any number of `np.ndarray`s, requiring this kind of annotation:
```suggestion
        self, batch_in: Tuple[np.ndarray, ...]
```

Reply:
Hmm, good point. I didn't know about this. I would lean towards creating a separate issue so that we can do it across the codebase.

Reply:
Fair. I'll create an issue.

Reply:
The issue is [here](https://github.com/RasaHQ/research/issues/150).